semgrep/tests/e2e/snapshots/test_rule_parser/test_rule_parser__failure__error_messages/badpattern/error.json

```diff
@@ -1,42 +1,11 @@
 {
   "errors": [
     {
-      "code": 4,
-      "level": "error",
-      "long_msg": "Pattern `$X == $X 3` could not be parsed as a Python semgrep pattern",
-      "short_msg": "invalid pattern",
-      "spans": [
-        {
-          "config_end": {
-            "col": 11,
-            "line": 0
-          },
-          "config_path": [
-            "rules",
-            0,
-            "patterns",
-            0,
-            "pattern"
-          ],
-          "config_start": {
-            "col": 1,
-            "line": 0
-          },
-          "context_end": null,
-          "context_start": null,
-          "end": {
-            "col": 24,
-            "line": 4
-          },
-          "file": "semgrep temp file",
-          "source_hash": "061f9b00eb8d4f4c3e2b8f6a4e4484ddadb637a33b943fdaa013532f23c7b212",
-          "start": {
-            "col": 14,
-            "line": 5
-          }
-        }
-      ],
-      "type": "InvalidPatternError"
+      "code": 2,
+      "level": "warn",
+      "message": "Semgrep Core WARN - Pattern parse error: In rule eqeq-is-bad: Invalid pattern for Python: Parsing.Parse_error",
```

Comment:
@emjin looks like message object returned by semgrep-core just has `Parsing.Parse_error` instead of the pattern that failed to parse?

Reply:
https://github.com/returntocorp/semgrep/issues/3860